demo/predictions/js/bpmn-diagram-bpmn-spec-pizza.js

```diff
@@ -0,0 +1,388 @@
+
+
+// inspired from the BPMN specification example 'triso - Order Process for Pizza V4.bpmn'
+function pizzaDiagram() {
```

Comment:
⚠️ To put in the file with all diagrams

Reply:
I don't want to do this. This diagram is only used here and putting all diagrams in the same file leads to a large file with a lot of unused content.
We should think about splitting the "all diagrams" file.